### PR TITLE
Add Visa.xml atmlocator exclusion (issue #4671)

### PR DESCRIPTION
**Pull request by jeremyn**
This should resolve issue #4671.

### CODE REVIEW

**Comment by J0WI** [comment]
Could you also add a comment about the issue?

File: src/chrome/content/rules/Visa.xml [hunk 1]
```diff
@@ -31,6 +31,10 @@
 	<!--securecookie host="^usa\.visa\.com$" name="^visaAnonCookie$" /-->
 
 
+	<exclusion pattern="^http://(?:www.)?visa\.com/atmlocator" />
```
Comment (J0WI) [inline src/chrome/content/rules/Visa.xml]: Prefer capturing groups (www\.)? over non-capturing (?:www\.)?.

**Comment by jeremyn** [comment]
I've changed the exclusion to use a capturing group, thanks.

I'm not sure what you mean about a comment.

**Comment by J0WI** [comment]
Just a comment like "page returns 404 error on https, see bug 4671"

**Comment by jeremyn** [comment]
I added a comment. The issue doesn't have extra information so I don't think linking to it is useful, and it's referenced in the first commit message anyway. If you feel strongly about linking to the issue in the ruleset, I'll modify the comment, just let me know.

**Comment by J0WI** [comment]
Rule lgtm, but this rule needs test coverage update.

**Comment by jeremyn** [comment]
I added tests but after checking around on the site, I think the ruleset needs to be reviewed generally.

**Comment by J0WI** [comment]
Of course this is just a specific bugfix without fully rebasing it, but it doesn't make the rule worse :)

**Pull request merged**